roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2

```diff
@@ -113,7 +113,7 @@ name="userPostSelfRegistration"
 subscriptions=["POST_ADD_USER"]
 
 [apim.throttling]
-service_url = "{{ api_devportal_server_url }}"
+service_url = "{{ traffic_manager_host_url }}/services/"
```

Comment:
Is `traffic_manager_host_url` initialised somewhere?

Reply:
It is in the 'dev/group_vars/apim.yml' file.

Reply:
ACK.